src/BlazorUI/Bit.BlazorUI.Extras/Styles/Fluent2/tokens.fluent2.scss

```
@@ -131,6 +131,8 @@
     --bit-layout-dialog-actions-direction: row;
     --bit-layout-dialog-actions-justify: flex-end;
     --bit-layout-dialog-actions-align: center;
+    // Fluent 2's dialog surface is 600px wide at its medium size.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)